pkg/applicationserver/applicationserver_test.go

```diff
@@ -66,6 +67,15 @@ type connChannels struct {
 	downErr     chan error
 }
 
+var testRights = []ttnpb.Right{
```

Comment:
Nit: These can sit in the `util_test` since they are used from all of the tests in the package. Same for other occurrences.

Reply:
will do